libdebug/cffi/ptrace_cffi_build.py

```diff
@@ -39,9 +39,13 @@
         // For details about the layout of the xsave structure, see Intel's Architecture Instruction Set Extensions Programming Reference
         // Chapter 3.2.4 "The Layout of XSAVE Save Area"
         // https://www.intel.com/content/dam/develop/external/us/en/documents/319433-024-697869.pdf
+        #pragma pack(push, 1)
```

Comment:
is not a bit slower? CP guy (guess who) told me so

Reply:
Also, why do you need it if you have padding? To force alignment with strange compilers?

Reply:
Without it the compiler might perform member reordering to try and optimize the access locations

Reply:
It's not slower, we are manually aligning it to 8 bytes and then telling the compiler not to touch it anymore